.delivery/build-cookbook/templates/default/Artifacts/MainTemplate.json.erb

```diff
@@ -34,7 +34,8 @@
       "type": "string"
     },
     "ssl_verify_mode": {
-      "type": "string"
+      "type": "string",
+      "defaultValue": "peer"
```

Comment:
I found that the `Microsoft.Common.OptionsGroup` element wasn't properly passing a default to the template so this is a hotfix for now.